pkg/deploy/graph/edges.go

```diff
@@ -73,6 +75,7 @@ func AddDeploymentEdges(g osgraph.MutableUniqueGraph, node *deploygraph.Deployme
 			}
 			if BelongsToDeploymentConfig(node.DeploymentConfig, rcNode.ReplicationController) {
 				g.AddEdge(node, rcNode, DeploymentEdgeKind)
+				g.AddEdge(node, rcNode, ControllerRefEdgeKind)
```

Comment:
@stevekuznetsov I will go ahead and open those changes in a new PR, unless it's better to add it in a new commit here? Thanks for the feedback on this btw